frontend/src/store/wallet.ts

```diff
@@ -216,7 +216,7 @@ export default function useWalletStore() {
       let connectedWallet;
       if (lastWallet.value) {
         [connectedWallet] = await onboard.value.connectWallet({
```

Comment:
(arbitrary comment)

The logo is now messed up. Do we have a svg version of the logo.